Doc/library/asyncio-eventloops.rst

```diff
@@ -165,7 +165,7 @@ An event loop policy must implement the following interface:
 
    .. method:: get_event_loop()
 
-      Get the event loop for the current context.
+      Get the event loop for the current coroutine.
```

Comment:
You want to say this not in the documentation of the interface, but in the documentation of the default policy.

Also, in the bpo issue I suggested adding a "Changed in 3.6" directive, but I don't see that here.